eve-tools/bpftrace-compiler/root/Dockerfile

```diff
@@ -25,7 +25,7 @@ COPY --from=eve-bpftrace /bpftrace/usr/bin/bpftrace /out/usr/bin/bpftrace
 COPY --from=eve-bpftrace /bpftrace-aotrt/ /out
 RUN /copy-out.sh /out/usr/bin/bpftrace /out
 
-FROM alpine:3.16 AS bpftrace-helper-compile
+FROM alpine:3.21 AS bpftrace-helper-compile
```

Comment:
just wondering why you build it in a separate alpine container... you could just build it inside eve-alpine in the previous step, since it already includes go by default